### PR TITLE
Revit_Core_Engine: Add Color convert

### LINKED ISSUE #1184: Add method to convert to System.Drawing color to Revit color
#### Description:
Need to add a new convert method to handle System.Drawing colors to Revit API colors.

### PR DESCRIPTION
**Pull request by tiagogrossi**
### Issues addressed by this PR

Closes #1184 

Color convert added, from System.Drawing to Revit API and vice-versa.

### Test files
N/A

### Changelog
Added `BH.Revit.Engine.Core.Convert.Color.ToRevit`
Added `BH.Revit.Engine.Core.Convert.Color.FromRevit`

### CODE REVIEW

**Comment by tiagogrossi** [comment]
@BHoMBot check compliance

File: Revit_Core_Engine/Convert/Revit/FromRevit/Color.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+namespace BH.Revit.Engine.Core
+{
+    public static partial class Convert
+    {
+        /***************************************************/
+        /****               Public Methods              ****/
+        /***************************************************/
+
+        [Description("Converts a Revit API color to a System.Drawing color.")]
+        [Input("color","The color to be converted")]
```
Comment (pawelbaran) [inline Revit_Core_Engine/Convert/Revit/FromRevit/Color.cs:36]:
```suggestion
        [Input("color","Revit color to be converted.")]
```

File: Revit_Core_Engine/Convert/Revit/ToRevit/Color.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+namespace BH.Revit.Engine.Core
+{
+    public static partial class Convert
+    {
+        /***************************************************/
+        /****               Public Methods              ****/
+        /***************************************************/
+
+        [Description("Converts a System.Drawing Color to a Revit API Color.")]
+        [Input("color","The color to be converted")]
```
Comment (pawelbaran) [inline Revit_Core_Engine/Convert/Revit/ToRevit/Color.cs:36]:
```suggestion
        [Input("color","System color to be converted.")]
```

**Review by pawelbaran — CHANGES_REQUESTED** [review]
I have no clue how did that happen that it has not been added yet - I am pretty sure that I spread a few instances of such convert across the repo, doing the convert manually each time 🙈 nice improvement. I would slightly tweak the descriptions and we are good to go!

**Review by pawelbaran — DISMISSED** [review]
Dots' lives saved - happy to approve now!

**Comment by pawelbaran** [comment]
@BHoMBot check required

**Comment by pawelbaran** [comment]
@BHoMBot check compliance
@BHoMBot check null-handling

**Comment by pawelbaran** [comment]
@BHoMBot check core
@BHoMBot check serialisation
@BHoMBot check versioning
@BHoMBot check installer

**Review by pawelbaran — APPROVED** [review]
Only minor tweak made since the last approval, re-approving now.

**Comment by FraserGreenroyd** [comment]
@BHoMBot this is a DevOps instruction. I am authorising dispensation to be granted on check ref. 5968602936

**Comment by pawelbaran** [comment]
@BHoMBot check ready-to-merge

**Comment by pawelbaran** [comment]
@BHoMBot check ready-to-merge

**Comment by pawelbaran** [comment]
@BHoMBot check ready-to-merge

**Comment by pawelbaran** [comment]
@BHoMBot check core

**Comment by pawelbaran** [comment]
@BHoMBot check ready-to-merge

**Comment by pawelbaran** [comment]
I am closing this PR and replacing it with the rebased version under https://github.com/BHoM/Revit_Toolkit/pull/1185

**Pull request closed without merging**